docs/documentation/models.rst

```diff
@@ -325,3 +325,15 @@ Wind chill temperature
 .. autoclass:: pythermalcomfort.classes_return.WCT
     :members:
 
+Work capacity
```

Comment:
Could you please change this heading to 
```
Work Capacity
===========
```

Below please briefly describe what these model are trying to predict.